<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainRecord</class>
 <widget class="QWidget" name="TrainRecord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1316</width>
    <height>1160</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>156</y>
     <width>1301</width>
     <height>981</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>1226</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #F1F9FD;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>131</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>训练日期</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>161</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>训练时长(min)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>10</y>
        <width>91</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>得分</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>10</y>
        <width>131</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>训练部位</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>10</y>
        <width>111</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>查看</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>10</y>
        <width>91</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QTableWidget" name="trainRecord_TableWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>1225</width>
       <height>781</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#trainRecord_TableWidget{border:none;}</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>870</y>
       <width>1221</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLineEdit" name="trainRecordDestPage_LineEdit">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>20</y>
        <width>91</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 90px;
height: 50px;
background: #FFFFFF;
border-radius: 4px;
border: 1px solid #EBEBEB;</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="trainRecordBackPage_btn">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>15</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/DependFile/Source/User/next.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_Btn">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>19</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/DependFile/Source/User/delete.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>22</y>
        <width>81</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>跳转到</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="allSelect_CheckBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>22</y>
        <width>80</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>全选</string>
      </property>
     </widget>
     <widget class="QPushButton" name="trainRecordFrontPage_btn">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>15</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/DependFile/Source/User/last.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="trainRecordIndexPage_Label">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>29</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>1/23</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="trainRecordGo_btn">
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>15</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #FFFFFF;
border-radius: 4px;
border: 1px solid #EBEBEB;</string>
      </property>
      <property name="text">
       <string>GO</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>24</y>
        <width>35</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>页</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createData_Btn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>93</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>生成数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="createNum_LineEdit">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1316</width>
     <height>1160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox_3{background:white;}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="portrait_Label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/DependFile/Source/User/user.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="name_Label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>34</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="trainRecord_Btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>102</y>
      <width>206</width>
      <height>54</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #05A6EC;
border-radius: 1px;
color:white;</string>
    </property>
    <property name="text">
     <string>训练记录</string>
    </property>
   </widget>
  </widget>
  <zorder>groupBox_3</zorder>
  <zorder>stackedWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
